pkg/sloop/ingress/kubewatcher_test.go

```diff
@@ -81,21 +82,58 @@ func Test_bigPicture(t *testing.T) {
 	masterURL := "url"
 	kubeContext := "" // empty string makes things work
 	enableGranularMetrics := true
-	kw, err := NewKubeWatcherSource(kubeClient, outChan, resync, includeCrds, time.Duration(10*time.Second), masterURL, kubeContext, enableGranularMetrics)
+	exclusionRules := map[string][]any{
```

Comment:
can we also add a test where exclusionRules are not provided, i.e. uses the default value of `ExclusionRules:           map[string][]any{},`

Reply:
I've added seperate tests for default and non-default exclusion rules